library/src/main/java/com/novoda/downloadmanager/DownloadBatchRequirementRules.java

```diff
@@ -3,7 +3,9 @@
 import java.util.ArrayList;
 import java.util.List;
 
-final class DownloadBatchRequirementRules implements DownloadBatchRequirementRule {
+import javax.annotation.Nullable;
+
+final class DownloadBatchRequirementRules implements DownloadBatchRequirementCompositeRule {
```

Comment:
The old class was named `DownloadBatchRequirementRules` so to keep the convention it should be just called
`DownloadBatchRequirementRules_Impl` and the interface should be called `DownloadBatchRequirementRules`

Reply:
They don't use the `Impl` convention (which I personally consider a bit iffy) anywhere in their codebase... I can submit the PR with such naming though, see what they say